tests/setup.sh

```diff
@@ -1,22 +1,27 @@
 #!/bin/bash
 
-# Setup the environment inside the test-image
-# DESTDIR is only for testing purposes, inside the image it's always /
-export DESTDIR=${1:-/}
+# Setup the environment in the test-image.
+# Runs INSIDE the image, with our newly built rpm in the path.
 
-mkdir -p $DESTDIR/build
-ln -sf ../data/SOURCES $DESTDIR/build/
+# Make sure nobody runs this accidentally outside the environment
+if [ ! /.rpmtestsuite ]; then
```

Comment:
Oops, this is missing an `-f`. Without it, the conditional is always false (i.e. the file isn't actually required to exist) :sweat_smile:

It's funny because this is the kind of thing that you just don't want to test locally :smile:

Reply:
Eh, this is why you never ever commit changes late in Friday afternoon :laughing: What a brainfade...